source/wp-content/themes/wporg-documentation-2022/parts/search.html

```diff
@@ -1 +1 @@
-<!-- wp:search {"label":"Search documentation","showLabel":false,"width":100,"widthUnit":"%","buttonText":"Search","buttonPosition":"button-inside","buttonUseIcon":true,"className":"is-style-secondary-search-control","placeholder":"Search documentation for…"} /-->
+<!-- wp:search {"label":"Search documentation","showLabel":false,"width":232,"widthUnit":"px","buttonText":"Search","buttonPosition":"button-inside","buttonUseIcon":true,"className":"is-style-secondary-search-control","placeholder":"Search documentation for…"} /-->
```

Comment:
This cuts off the label on Firefox:

<img width="456" alt="Screenshot 2024-06-10 at 5 46 01 PM" src="https://github.com/WordPress/wporg-documentation-2022/assets/541093/2653e934-da1d-4ac0-b494-01ea2cbf4447">

(I'm good with shipping the PR and making an issue for this, just wanted to flag it)

Reply:
Fixed in [de7bc81](https://github.com/WordPress/wporg-documentation-2022/pull/97/commits/de7bc81babf54d943c6c67cba4258d81750bb0f2)